erroranalysis/requirements-dev.txt

```diff
@@ -6,3 +6,4 @@ requests==2.25.1
 requirements-parser==0.2.0
 xgboost<=1.0.0
 shap>=0.20.0, <=0.39.0
+pytest-faulthandler
```

Comment:
Do we need to add this dependency at all since looks like pytest >=5.0.0 should already have this capability?

https://pypi.org/project/pytest-faulthandler/

This plugin is now part of pytest core since pytest 5.0, so users should not install this plugin together with that pytest version.

Version 2.0.0 of this package is a dummy package for pytest 5.0 compatibility.

Reply:
oh, interesting, I didn't know it was included by default.
Looking at the build based on this PR:
https://github.com/microsoft/responsible-ai-toolbox/runs/6082137912?check_suite_focus=true
unfortunately I don't see any extra information...